utils/cleanup_mocked_prometheus_response.py

```diff
@@ -16,46 +16,60 @@ def extract_property(array_of_lines, propertyName):
 
 def replace_values(lines):
     result = []
-    replacements = [
-        ('node', 'test-node'),
-        ('instance', 'test-node'),
-        ('namespace', 'test-namespace'),
-        ('daemonset', 'test-daemonset'),
-        ('deployment', 'test-deployment'),
-        ('statefulset', 'test-statefulset'),
-        ('replicaset', 'test-replicaset'),
-        ('pod', 'test-pod'),
-        ('service', 'test-service'),
-        ('container', 'test-container'),
-        ('job', 'test-job'),
-        ('job_name', 'test-job-name'),
+    replacements: list[tuple[str, str, str | None]] = [
```

Comment:
The whole script was changed to take the first datapoint for each found metric matching the set criteria. In addition to the previously existing criteria, it now ensures that if there are datapoints with attributes `container="POD"` and `container=".+"`, both datapoints are in the output dump. This is to enable a unit test in `tests/integration/test_metric_collection.py`.